pkg/controller/instance/instance_controller.go

```diff
@@ -67,34 +67,33 @@ type Reconciler struct {
 // SetupWithManager registers this reconciler with the controller manager
 func (r *Reconciler) SetupWithManager(
 	mgr ctrl.Manager) error {
-	addOvRelatedInstancesToReconcile := handler.ToRequestsFunc(
```

Comment:
not sure who this is tied to 1.18 bump

Reply:
I went back through my notes and I'm honestly not sure what required this change. I don't think it was the 1.18 bump though.

I'm pretty sure it was the bump in the controller-runtime library that caused it. It doesn't look like handler.ToRequestsFunc exists in the bumped version.